content/millennium/dstu2/general-clinical/condition.md

```diff
@@ -131,40 +215,55 @@ Create Diagnosis Example:
 
 The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
 
+### Errors
+
+The common [errors] may be returned. In addition, [OperationOutcomes] may be returned in the following scenarios:
+                                                                        
+ HTTP Status | Cause                              | Severity  | Code
+-------------|------------------------------------|-----------|---------------
+ 422         | Body contained modifier extensions | error     | extension
+ 422         | Body contained implicit rules      | error     | unsupported
+
+
 ## Update
 
 Update an existing condition.
 
     PUT /Condition/:id
 
-Note that any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+_Implementation Notes_
 
-### Headers
+* Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+* Currently, `health-concern` category code is not supported for updating conditions.
 
-To successfully PUT a condition, the following headers must be provided. Condition updates are supported from the closed endpoint only.
+### Authorization Types
 
-    Content-Type: application/json+fhir
-    Authorization: <OAuth2 Bearer Token>
-    If-Match: W/"<Current condition id>"
+<%= authorization_types(practitioner: true, system: true)%>
 
-_Implementation Notes_
+### Headers
 
-* Currently, `health-concern` category code is not supported for updating conditions.
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir', 
+                   'Content-Type': 'application/json+fhir', 'If-Match': 'W/"&lt;Current version of the Condition resource>"'} %>
 
 ### Body fields
 
+Notes:
+
+* abatementDateTime is not supported for Conditions with a category of `diagnosis`.
```

Comment:
Should we put abatementDateTime in the quotes like diagnosis to make it stand out?